--- sysvinit-2.87dsf/src/Makefile.orig	2011-07-26 00:35:31.682000820 +0900
+++ sysvinit-2.87dsf/src/Makefile	2011-07-26 00:36:45.422000845 +0900
@@ -47,8 +47,8 @@
 BIN_GROUP	= root
 BIN_COMBO	= $(BIN_OWNER):$(BIN_GROUP)
 STRIP		= strip -s -R .comment
-INSTALL_EXEC	= install -o $(BIN_OWNER) -g $(BIN_GROUP) -m 755
-INSTALL_DATA	= install -o $(BIN_OWNER) -g $(BIN_GROUP) -m 644
+INSTALL_EXEC	= install -m 755
+INSTALL_DATA	= install -m 644
 MANDIR		= /usr/share/man
 
 ifeq ($(WITH_SELINUX),yes)
